app/helpers/sellers_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module SellersHelper
```

Comment:
This module has tests that you're going to have to update — also I have a feeling that some other adjacent tests will fail 😅

Reply:
i checked! didn't see anything and ctrl + f didn't see any location test errors besides the one in the helper